terraform/eks/daemon/main.tf

```diff
@@ -100,6 +101,11 @@ resource "aws_iam_role_policy_attachment" "node_CloudWatchAgentServerPolicy" {
   role       = aws_iam_role.node_role.name
 }
 
+resource "aws_iam_role_policy_attachment" "node_AmazonEKSClusterPolicy" {
```

Comment:
Question - do we need the cluster policy on this IAM role? The node role is attached to each node of the cluster. The purpose (as I know it) of the AmazonEKSClusterPolicy managed policy is it is for managing the cluster.

Reply:
Honestly, I added this to try and resolve some of the issues with the iam roles, but I don't think this was actually needed. Can't say for sure as all my testing was done with it present, but I think it would be fine to remove it and if need be, we can add it back

Reply:
removed, will add back if it is determined to be necessary